src/utils/constants.ts

```diff
@@ -1,59 +1,104 @@
 import { BigNumber } from 'ethers'
 
+type ChainConfig = {
+  router: string
+  creationBlock: number
+  weth: string
+}
+
+const WETH_NOT_SUPPORTED_ON_CHAIN = '0x0000000000000000000000000000000000000000'
```

Comment:
Made this name long on purpose so that the linting of the map is pretty 😂  with a shorter name some chains go onto a single line